devices/cpu/registers.h

```diff
@@ -55,11 +76,20 @@ typedef struct Cpu {
   Usci *usci;
   Bcm *bcm;
   Timer_a *timer_a;
+
+  CallTracer callTracer;
+  CpuStats stats;
 } Cpu;
 
-Status_reg get_sr_fields (Emulator *emu);
-void set_sr_from_fields(Emulator *emu, const Status_reg fields);
-void initialize_msp_registers (Emulator *emu);
-void update_register_display (Emulator *emu);
+Status_reg get_sr_fields (Emulator* const emu);
```

Comment:
I prefer const Emulator* over Emulator* const. This also refers to other const pointers.

Reply:
well, it's not matter of preference, but meaning :D 
`Emulrator* const emu` - `emu` is const, but pointed `Emulator` can be changed
`Emulator const* emu` - pointed `Emulator` can't be changed, but `emu` can

Reply:
True, my mistake.